src/java.base/share/classes/java/net/InetSocketAddress.java

```diff
@@ -308,6 +321,10 @@ private void readObject(ObjectInputStream in)
         UNSAFE.putReference(this, FIELDS_OFFSET, h);
     }
 
+    /**
+     * Throws {@code InvalidObjectException}, always.
+     * @throws ObjectStreamException always
```

Comment:
Is this correct? Seems to contradict each other.

Reply:
`InvalidObjectException` is a subclass of `ObjectStreamException`.